src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -852,7 +862,21 @@ export class RbdFormComponent extends CdForm implements OnInit {
   }
 
   shouldDisable(option: string): boolean {
-    return this.currentPoolMirrorMode === 'pool' && option === 'snapshot' ? true : null;
+    return this.currentPoolMirrorMode === this.rbdPoolMirrorModes.pool &&
+      option === this.rbdMirrorModes.snapshot
+      ? true
+      : false;
+  }
+
+  disableMirroring(namespace: string) {
+    const mirroringControl = this.rbdForm.get('mirroring');
+    if (namespace) {
+      this.mirroring = false;
```

Comment:
I looked at demo, the mirroring controls are getting disabled but the mirroring is not set to false. The Disabled Checked Tickbox is showing up, when we are selecting namespace.